frontend/common/src/components/inputPartials/Fieldset/Fieldset.tsx

```diff
@@ -44,13 +44,22 @@ export const Fieldset: React.FC<FieldsetProps> = ({
       }}
       data-h2-margin="b(bottom, xxs)"
     >
+      <legend data-h2-visibility="b(invisible)" data-h2-font-size="b(caption)">
```

Comment:
I'm definitely not the expert here but is the font size attribute here doing anything?  It seems confusing the make something invisible and then set a font size but maybe it has some significance that I'm not thinking of?

Reply:
No, it doesn't do anything useful outside of inflating the DOM. I was more worried about hiding it and forgot to remove the, now unnecessary, font size rule.

Resolved with: [remove unnecessary font-size rule](https://github.com/GCTC-NTGC/gc-digital-talent/pull/2429/commits/0e41f58655512200705c065bdcf42a30e5493525)